test/rake_helper.rb

```diff
@@ -3,7 +3,7 @@
 POSTGIS_TEST_HELPER = "test/test_helper.rb"
 
 def ar_root
-  File.join(Gem.loaded_specs["rails"].full_gem_path, "activerecord")
+  Gem.loaded_specs["activerecord"].full_gem_path
```

Comment:
Does this allow the activerecord tests to get loaded in without installing the gem directly?

Reply:
This is just a direct reference to the actual gem folder.